examples/rust/raw_mesh/Cargo.toml

```diff
@@ -9,7 +9,7 @@ publish = false
 [dependencies]
 rerun = { path = "../../../crates/rerun", features = [
   "native_viewer",
-  "web_viewer",
+  # "web_viewer",
```

Comment:
maybe we should just remove this feature flag? 😬

Reply:
Eh, it's quite nice to show off the browser support; it's only annoying for us developers...